src/elements/ExpiryIndicator.vue

```diff
@@ -0,0 +1,96 @@
+<script setup lang="ts">
+import ExpiryIcon from '@/icons/ExpiryIcon.vue';
+import {computed} from "vue";
+import { t } from '@/composable/i18n';
+
+// component properties
+interface Props {
+  days?: Integer;
```

Comment:
Can you adjust it so we can define a unit? 

Maybe:
Seconds, Minutes, Hours, Days, Weeks, Months, and Years? 

And an enum being defined that could be imported and used in https://github.com/thunderbird/services-ui/blob/main/src/definitions.ts

And then have it pull in the correct localization for the specific unit.

Thanks!